dev-docs/bidders/spotx.md

```diff
@@ -12,24 +12,39 @@ biddercode: spotx
-| `hide_skin`      | optional | Set to true to hide the spotx skin                                                                                                                            |                         | `boolean` |
-| `autoplay`       | optional | Set to true to attempt to play the ad as soon as it is loaded                                                                                                 |                         | `boolean` |
-| `ad_mute`        | optional | Set to true to start the ad with the volume muted                                                                                                             |                         | `boolean` |
-| `custom`         | optional | See SpotX documentation [here](https://developer.spotxchange.com/content/local/docs/sdkDocs/DirectSdk/README.md#custom-property-for-key-value-pair-reporting) |                         | `object`  |
+| Name                 | Scope    | Description                                                                                                                                                   | Example                                                                                                                                                                                              | Type        |
+|----------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|
+| `channel_id`         | required | A unique 5 digit ID that is generated by the SpotX publisher platform when a channel is created                                                               | `'85394'`                                                                                                                                                                                            | `string`    |
+| `ad_unit`            | required | Token that describes which ad unit to play: instream or outstream                                                                                             | `'outstream'`                                                                                                                                                                                        | `string`    |
+| `outstream_function` | optional | Custom function to be used as a renderer.                                                                                                                     | `function(bid){console.log(bid);}`                                                                                                                                                                   | `function`  |
```

Comment:
What is the bid input parameter?

Reply:
This is the bid parameter that is passed to all renderers: http://prebid.org/dev-docs/show-outstream-video-ads.html#renderers